Kinesis blog post

gunnarmorling opened the pull request:
@jpechane Feedback welcome :)

jpechane commented on blog/2018-08-30-streaming-mysql-data-changes-into-kinesis.adoc:
```diff
@@ -0,0 +1,411 @@
+For this example we're simply going to keep the offsets and database history in memory.
+I.e. if the engine is restarted, this information will be lost and the connector will start from scratch, e.g. with a new initial snapshot.
+
+While out of scope for this blog post, it wouldn't be too difficult to create alternative implementations of the `OffsetBackingStore` and `DatabaseHistory` contracts, respectively.
```
I think file based stores can get honorable mention as they are available for limit ed produciton use cases

gunnarmorling replied:
Ok, will do.

jpechane commented on blog/2018-08-30-streaming-mysql-data-changes-into-kinesis.adoc:
```diff
@@ -0,0 +1,411 @@
+    message.put("value", record.value());
+
+    // create partition key by hashing the record's key
+    String partitionKey = String.valueOf(record.key() != null ? record.key().hashCode() : new Object().hashCode());
```
Should not all records without key end up in the same partition?

gunnarmorling replied:
Yes, good thinking. I'll set it to a fixed value.

jpechane commented on blog/2018-08-30-streaming-mysql-data-changes-into-kinesis.adoc:
```diff
@@ -0,0 +1,411 @@
+    String partitionKey = String.valueOf(record.key() != null ? record.key().hashCode() : new Object().hashCode());
+
+    // create data blob representing the container by using Kafka Connect's JSON converter
+    final byte[] payload = valueConverter.fromConnectData("dummy", schema, message);
```
I think this does not correspond to text - only value is sent, is not it?

jpechane commented:
@gunnarmorling Very nice and easy to follow and understand! I've just added one note and otherwise we are good to go.

gunnarmorling commented:
Addressed your remarks, applied a few more changes and merged. Thanks again for reviewing and your help, @jpechane! Btw. the broken rendering was caused apparently by an extra-whitespace after one of the post tags in the array within the document's pre-amble. Go figure...
